test/redis/store/marshalling_test.rb

```diff
@@ -23,6 +23,12 @@ def teardown
     @store.get("rabbit").must_equal(@white_rabbit)
   end
 
+  it "marshals on multi set" do
+    @store.mset "rabbit", @white_rabbit, "rabbit2", @rabbit
```

Comment:
I would say that until you have the need for it, there's no need to add any code. As far as you or I know, you're the only person that will ever need to use `mset`. Plus, if someone needs to add it later on, they can just copy your implementation.